test/plugin/scenarios/okhttp-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,137 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+registryItems:
+  applications:
+  - {okhttp-scenario: nq 0}
+  instances:
+  - {okhttp-scenario: 1}
+  operationNames:
+  - okhttp-scenario: [/case/healthCheck, /case/okhttp-case, /case/receiveContext-0]
+  heartbeat: []
+segmentItems:
+- applicationCode: okhttp-scenario
+  segmentSize: 5
+  segments:
+  - segmentId: not null
+    spans:
+    - operationName: /case/healthCheck
+      operationId: 0
+      parentSpanId: -1
+      spanId: 0
+      spanLayer: Http
+      startTime: nq 0
+      endTime: nq 0
+      componentId: 14
+      componentName: ''
+      isError: false
+      spanType: Entry
+      peer: ''
+      peerId: 0
+      tags:
+        - {key: url, value: 'http://localhost:8080/okhttp-case/case/healthCheck'}
+        - {key: http.method, value: HEAD}
```

Comment:
We don't need to verify `/case/healthCheck`. 
Because it will be called several times when `healthcheck` failure.

Reply:
That's tell me i can delete the healthCheck in the expectedData.xml?

Reply:
I've delete the healthcheck in xml. @dmsolr